### PR TITLE
Remove old repository name

### PR DESCRIPTION
**Pull request by TerryHowe**
Remove old repository name from docs.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/contents/users/build.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ make docker-push
 ## Build Remaining Containers
 
 The second step of the build is to build the rest of the containers out
-of the [EKS Distro Build Steps Repository](https://github.com/aws/eks-distro).
+of the [EKS Distro Container Repository](https://github.com/aws/eks-distro).
```
Comment (jaxesn) [inline docs/contents/users/build.md:34]: Should it just be `EKS Distro Repository`, without the Container?

Comment (TerryHowe) [reply docs/contents/users/build.md:34]: Yeh, that is what we call it and I'm fine with that, but I thought this name was a bit more descriptive

Comment (micahhausler) [reply docs/contents/users/build.md:34]: I'm inclined for `EKS Distro Repository`

Comment (TerryHowe) [reply docs/contents/users/build.md]: Done

**Review by micahhausler — APPROVED** [review]
/lgtm
/approve

**Pull request merged**